File: src/main/java/com/example/member/dto/MemberDto.java
package com.example.member.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class MemberDto {
        private String loginId;
        private String password;
        private String userName;
        private String email;
        private int gender;
        private String birthday;
        private String cellPhone;
        private int status;
}

File: src/main/kotlin/com/example/member/dto/MemberDto.kt
package com.example.member.dto

import lombok.NoArgsConstructor

@NoArgsConstructor
class MemberDto(
    var loginId: String,
    var password: String,
    var userName: String,
    var email: String,
    var gender: Int,
    var birthday: String,
    var cellPhone: String,
    var status: Int
)
